test/fuzz/fuzzer_pass_donate_modules_test.cpp

```diff
@@ -270,7 +270,8 @@ TEST(FuzzerPassDonateModulesTest, DonationWithUniforms) {
 
   FactManager fact_manager;
 
-  FuzzerContext fuzzer_context(MakeUnique<PseudoRandomGenerator>(0).get(), 100);
```

Comment:
This is undefined behaviour. `std::unique_ptr::get` returns a pointer to the data that `unique_ptr` manages. Then `unique_ptr` gets deallocated since it's a rvalue and we are left with a dangling pointer. Solution is to use `unique_ptr::release` or simply not to use `unique_ptr` at all ;)

Reply:
Good catch - thanks for fixing.